### PR TITLE
[14.0][MIG] shopinvader_guest_mode

### PR DESCRIPTION
**Pull request by acsonefho**
Migration of module `shopinvader_guest_mode`
Initally done into this PR https://github.com/shopinvader/odoo-shopinvader/pull/952 but I don't have write access.

### CODE REVIEW

File: shopinvader_guest_mode/__manifest__.py [hunk 1]
```diff
@@ -3,11 +3,10 @@
 
 {
     "name": "Shopinvader Guest Mode",
-    "summary": """
-        Guest mode for Shopinvader""",
-    "version": "13.0.1.1.0",
+    "summary": """Guest mode for Shopinvader""",
+    "version": "14.0.1.1.0",
```
Comment (ivantodorovich) [inline shopinvader_guest_mode/__manifest__.py]:
```suggestion
    "version": "14.0.1.0.0",
```

File: shopinvader_guest_mode/models/shopinvader_partner.py [hunk 2]
```diff
@@ -60,10 +61,8 @@ class ShopinvaderPartner(models.Model):
         ("email_uniq", "CHECK(1=1)", ""),
         (
             "unique_shopinvader_partner_email",
-            "EXCLUDE (backend_id WITH =, partner_email WITH =)"
-            " WHERE (active=True)",
-            "Only one active binding with the same email is "
-            "allowed by backend.",
+            "EXCLUDE (backend_id WITH =, partner_email WITH =)" " WHERE (active=True)",
+            "Only one active binding with the same email is " "allowed by backend.",
```
Comment (ivantodorovich) [inline shopinvader_guest_mode/models/shopinvader_partner.py]:
```suggestion
            "EXCLUDE (backend_id WITH =, partner_email WITH =) WHERE (active=True)",
            "Only one active binding with the same email is allowed by backend.",
```

**Review by ivantodorovich — APPROVED** [review]
minor comments

**Review by rousseldenis — APPROVED** [review]
Code review

**Comment by rousseldenis** [comment]
@acsonefho Ok, we merge this as you took commits from @david-popa.

/ocabot merge patch

**Comment by rousseldenis** [comment]
@acsonefho Could you check this as shopinvader config has moved to res.config.settings to be more standard?

**Pull request merged**